apps/emqx_management/src/emqx_mgmt_api_clients.erl

```diff
@@ -669,3 +709,24 @@ format_authz_cache({{PubSub, Topic}, {AuthzResult, Timestamp}}) ->
        result => AuthzResult,
        updated_time => Timestamp
      }.
+
+%%--------------------------------------------------------------------
+%% time format funcs
+
+unix_ts_to_rfc3339_bin(undefined) ->
+    <<"undefined">>;
```

Comment:
is this possible ? line 693 seem to have `undefined` covered ?

Reply:
It's covered. Also the in parameter `timestamp` and `rfc3339` would never be `'undefined'` in this api.
So it's not necessary in this module. Just defensive code for generic `emqx_rfc3339` later.
Should the two funcs be deleted in the module?

Reply:
the two funs would be removed. _"Occam's Razor"_